tests/test_crawlers.py

```diff
@@ -391,6 +406,21 @@ def test_intersects_time_range_no_upper_limit(self):
         self.assertFalse(crawler._intersects_time_range(None, datetime(2019, 2, 19)))
 
 
+class URLNavigationTestCase(unittest.TestCase):
+    @mock.patch("geospaas_harvesting.crawlers.OpenDAPCrawler._get_links")
+    @mock.patch("geospaas_harvesting.crawlers.OpenDAPCrawler._http_get")
+    def test_correct_navigation_to_download_page_for_OSISAF(self, mock_get_link, mock_http_get):
```

Comment:
Yes, it does not matter, because I want the result of this line to be based on my desire:
https://github.com/nansencenter/django-geo-spaas-harvesting/blob/e350993f66fa99aeaf1093d152a87cd59b30f0b1/geospaas_harvesting/crawlers.py#L277
And The test concerns **the rest of the function** not **this line**.

Reply:
@aperrin66 
I have not seen your next comment. Let me fix it now!

Reply:
Done in the next commit